.github/workflows/ci.yml

```
@@ -1,785 +1,273 @@
-        uses: actions/cache@v5
-        continue-on-error: true
+        id: cache-qt
+        uses: actions/cache@0057852bfaa89a56745cba8c7296529d2fc39830 # v4
         with:
-          path: ${{ runner.workspace }}/Qt
-          key: windows-qt-6.10.0-win64_msvc2022_64-qtmultimedia-qtshadertools-qtimageformats-qt5compat-v2
-          restore-keys: |
-            windows-qt-6.10.0-win64_msvc2022_64-qtmultimedia-qtshadertools-qtimageformats-qt5compat-
-
-      - name: Install aqtinstall
+          path: ${{ runner.workspace }}\Qt
```

actionlint (1.7.12)

[error] property "workspace" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)